packages/edit-site/src/components/global-styles/ui.js

```diff
@@ -191,6 +192,20 @@ function GlobalStylesStyleBook() {
 				)
 			}
 			onSelect={ ( blockName ) => {
+				if ( blockName === 'duotones' ) {
+					return;
```

Comment:
In the case of duotone, what do you reckon about adding a CSS rule to `#example-duotones` to remove the `cursor: pointer` so it doesn't behave like a link?

Reply:
Sounds like a plan 👍

Reply:
I took a slightly different approach to effectively disabling the duotone examples in https://github.com/WordPress/gutenberg/pull/65692/commits/d4ed58a2ace82be35b1c3915f34ea212d1b38419. Let me know if you think I've gone off the rails there or if it's something we can refine.

Reply:
Nice! Disabling is way better, removes the selected border as well. I like!



https://github.com/user-attachments/assets/45855717-4484-4b04-8445-240200e32f8e

Reply:
I believe it should still be accessible even though it's disabled for keyboard navigation/discovery, assistive tech etc. As a follow-up, I think it would be good to get some broader a11y reviews of the Style Book once the dust settles.